File: MyRadio2/app/src/main/java/com/example/myradio2/RadioStation.java
package com.example.myradio2;

public class RadioStation {
    private String name;
    private String country;
    private String streamUrl;

    public RadioStation(String name, String country, String streamUrl) {
        this.name = name;
        this.country = country;
        this.streamUrl = streamUrl;
    }

    public String getName() {
        return name;
    }

    public String getCountry() {
        return country;
    }

    public String getStreamUrl() {
        return streamUrl;
    }
}

File: WorldRadio/app/src/main/java/com/example/worldradio/data/model/RadioStation.kt
package com.example.worldradio.data.model

import com.google.gson.annotations.SerializedName
/**
 * 라디오 방송국 데이터 모델
 */
data class RadioStation(
    val stationuuid: String,
    val name: String,
    val country: String?,
    val favicon: String?,
    val url: String,
    val urlResolved: String?
)